src/languages/vbnet.js

```diff
@@ -1,63 +1,114 @@
+        'join key let lib loop me mid module mustinherit mustoverride mybase myclass ' + /* j-m */
+        'namespace narrowing new next notinheritable notoverridable ' + /* n */
+        'of off on operator option optional order overloads overridable overrides ' + /* o */
         'paramarray partial preserve private property protected public ' + /* p */
-        'raiseevent readonly redim rem removehandler resume return ' + /* r */
+        'raiseevent readonly redim removehandler resume return ' + /* r */
         'select set shadows shared skip static step stop structure strict sub synclock ' + /* s */
-        'take text then throw to try unicode until using when where while widening with withevents writeonly xor yield', /* t-y */
+        'take text then throw to try unicode until using when where while widening with withevents writeonly yield' /* t-y */,
       built_in:
-        'boolean byte cbool cbyte cchar cdate cdec cdbl char cint clng cobj csbyte cshort csng cstr ctype ' +  /* b-c */
-        'date decimal directcast double gettype getxmlnamespace iif integer long object ' + /* d-o */
-        'sbyte short single string trycast typeof uinteger ulong ushort', /* s-u */
-      literal:
-        'true false nothing'
+        // Operators https://docs.microsoft.com/dotnet/visual-basic/language-reference/operators
+        'addressof and andalso await directcast gettype getxmlnamespace is isfalse isnot istrue like mod nameof new not or orelse trycast typeof xor ' +
+        // Type Conversion Functions https://docs.microsoft.com/dotnet/visual-basic/language-reference/functions/type-conversion-functions
+        'cbool cbyte cchar cdate cdbl cdec cint clng cobj csbyte cshort csng cstr cuint culng cushort',
+      type:
+        // Data types https://docs.microsoft.com/dotnet/visual-basic/language-reference/data-types
+        'boolean byte char date decimal double integer long object sbyte short single string uinteger ulong ushort',
+      literal: 'true false nothing',
     },
-    illegal: '//|\\{|\\}|endif|gosub|variant|wend|^\\$ ', /* reserved deprecated keywords */
+    illegal:
+      '//|\\{|\\}|endif|gosub|variant|wend|^\\$ ' /* reserved deprecated keywords */,
     contains: [
-      hljs.inherit(hljs.QUOTE_STRING_MODE, {contains: [{begin: '""'}]}),
-      hljs.COMMENT(
-        '\'',
-        '$',
-        {
-          returnBegin: true,
-          contains: [
-            {
-              className: 'doctag',
-              begin: '\'\'\'|<!--|-->',
-              contains: [hljs.PHRASAL_WORDS_MODE]
-            },
-            {
-              className: 'doctag',
-              begin: '</?', end: '>',
-              contains: [hljs.PHRASAL_WORDS_MODE]
-            }
-          ]
-        }
-      ),
-      hljs.C_NUMBER_MODE,
-      {
-        className: 'meta',
-        begin: '#', end: '$',
-        keywords: {'meta-keyword': 'if else elseif end region externalsource'}
-      }
-    ]
+      CHAR,
```

Comment:
```suggestion
      CHARACTER,
```

Prefer to use full words for readability vs abbreviations. (obivously need to change the top also)